client/asset/eth/multirpc.go

```diff
@@ -113,7 +113,9 @@ func (p *provider) cachedTip() *types.Header {
 
 	p.tip.RLock()
 	defer p.tip.RUnlock()
-	if time.Since(p.tip.failStamp) < failQuarantine || time.Since(p.tip.headerStamp) > stale {
+	if time.Since(p.tip.failStamp) < failQuarantine ||
+		time.Since(p.tip.headerStamp) > stale ||
+		time.Now().Unix()-int64(p.tip.header.Time) > dexeth.MaxBlockInterval {
```

Comment:
I got the impression on matrix that @buck54321 is going to do some things around here.

Reply:
Removing the stale check.

Reply:
Or, actually,  with no stale check it only fetches a new header after `dexeth.MaxBlockInterval`... So maybe one block interval after that should fetch a new one.

Reply:
After seeing it now, I think maybe we did need the `stale` check. It's the `MaxBlockInterval` check that could have been deleted, since `stale` was always more restrictive. But we definitely don't want to check after 13 seconds when we have a websocket connection.

Reply:
Put it back. The header can still be 180 seconds old though. Thats a few blocks.